app/templates/direct-award/view-project.html

```diff
@@ -0,0 +1,40 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Save your search - Digital Marketplace{% endblock %}
```

Comment:
"Saved search"?

Reply:
:man_shrugging: I've fixed this in a PR that sits on top of this already (because you know, design to be done later), but I'll pull it in here anyway.